frontend/src/utils/account-send.ts

```diff
@@ -27,7 +28,7 @@ type AccountDataToEncrypt = {
 
 // Used to create an encryption key which encrypts account send data
 type KeyData = {
-  encryptionCount: number;
+  encryptionCount: string;
```

Comment:
`BigNumber` or `bigint` would be better types here since this is an integer

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/494/commits/34e3357dfa694fa00ae1706a4e8ac520e4cceb48 updated here